multiworld/core/flat_goal_env.py

```diff
@@ -42,6 +42,7 @@ def __init__(self, wrapped_env, obs_keys=None, goal_keys=None):
                 for k in goal_keys
             ]),
         )
+        self.obs_dim = self.observation_space.low.size
```

Comment:
It's a bit risky to add new things to the interface, because now if our code starts assuming that `env.obs_dim` exists, then our code won't be compatible with other `gym` envs. This is something that came up a lot with `rllab`, so I'd like to avoid having this issue in our code.

Reply:
This ended up actually only being temporary code that I forgot to delete, but good to know!